src/cursor/cur_index.c

```diff
@@ -349,13 +349,15 @@ static int
 __curindex_open_colgroups(
     WT_SESSION_IMPL *session, WT_CURSOR_INDEX *cindex, const char *cfg_arg[])
 {
+	WT_CONFIG_ITEM cval;
 	WT_TABLE *table;
 	WT_CURSOR **cp;
-	u_long arg;
+	char cfg_bulk[30], *proj;
 	/* Child cursors are opened with dump disabled. */
-	const char *cfg[] = { cfg_arg[0], cfg_arg[1], "dump=\"\"", NULL };
```

Comment:
@agorrod If `bulk=unordered` is not removed for column group files, then `test_backup01` fails. If `bulk=unordered` only is ever intended for index insertion, then I agree that column group files should not be opened. Can we follow this up as a separate issue?